theano/tensor/blas.py

```diff
@@ -1192,43 +1192,43 @@ def _beta_L_plus_alpha_M(beta, L, alpha, M, recurse_flip=True):
 
     # this is False'd out because of inadequate testing.
     # TODO see ticket #237
-    if False and res_is_a(M, gemm_no_inplace, 1):
-        # EXPRESSION: (beta * L) + (alpha * (gemm_no_inplace(G, a, u, v, b)))
-        # EXPRESSION: (beta * L) + alpha * (b * G) + alpha * a * dot(u, v)
-        G, a, u, v, b = M.owner.inputs
-        # print 'GEMM', G, L
-
-        if res_is_a(G, _dot22, 1):
-            # EXPRESSION: (beta * L) +
-            #            (alpha * (gemm_no_inplace(dot(x,y), a, u, v, b)))
-            x, y = G.owner.inputs
-
-            # EXPRESSION: (beta * L) + (alpha * ((b*dot(x,y) +
-            #            (a * dot(u, v)))))
-            # EXPRESSION: (beta * L) + (alpha*b*dot(x,y)) +
-            #            (alpha * a * dot(u, v))
-            rval = [
-                gemm_no_inplace(
-                    gemm_no_inplace(L, alpha * b, x, y, beta), alpha * a, u, v, 1.0
-                )
-            ]
-            return rval
-        if G is L:
-            # EXPRESSION: (beta * L) + (alpha*b*L) + (alpha * a * dot(u, v))
-            rval = [gemm_no_inplace(L, alpha * a, u, v, alpha * b + beta)]
-            return rval
-        if 1.0 != alpha:
-            # at the very least, move the alpha inside the gemm_no_inplace
-            rval = [beta * L + gemm_no_inplace(G, alpha * a, u, v, alpha * b)]
-            return rval
+    # if res_is_a(fgraph, M, gemm_no_inplace, 1):
```

Comment:
Shouldn't we just remove this?

Reply:
Yeah, I only kept it around in case someone wanted to figure out what this was about and&mdash;possibly&mdash;get it working.